private/pkg/thread/thread.go

```diff
@@ -77,7 +77,6 @@ func Parallelize(ctx context.Context, jobs []func(context.Context) error, option
 		if stop {
 			break
 		}
-		job := job
```

Comment:
i would keep this here, only doing the set within `default` feels like a nasty bug in the future if we ever change this and refer to `job` somewhere else

Reply:
if anything id move it above the `if stop {`, always keeping it close to the `for` loop start

Reply:
I moved it closer to the usage to avoid declaring it on each loop w/out any usage. If using it elsewhere, pretty sure govet would lint it, and also this loop scope var issue it's supposed to [be fixed soon by the go team](https://github.com/golang/go/issues/60078). I'll move it back up, although I don't think it's quite necessary.

Reply:
oh fair enough...idk dont worry about it